rgw/v2/tests/s3_swift/reusable.py

```diff
@@ -66,6 +66,29 @@ def create_bucket(bucket_name, rgw, user_info, location=None):
     return bucket
 
 
+def create_bucket_readonly(bucket_name, rgw, user_info):
+    log.info("creating bucket with name: %s" % bucket_name)
+    # bucket = s3_ops.resource_op(rgw_conn, 'Bucket', bucket_name_to_create)
```

Comment:
please remove commented line

Reply:
@ckulal resolved , can you please review now?